source/content/guides/composer-convert.md

```diff
@@ -169,19 +214,31 @@ If you are using an exported config, you will need to move the configuration fil
 
 Locate the configuration files in your existing site and move them here. If they are stored in the files directory on your existing site, retrieve them via [SFTP](/sftp), as the Git clone would not contain them. The example project is configured to use this location.
 
-## Update to Latest Drupal Core
+## Set up a Multidev (Optional)
 
-The Pantheon Drupal 8 Composer repository may not depend on the latest secure release of Drupal Core. To keep your site safe, it's a good idea to pull the latest version in now -- before moving on -- by running:
+If your Pantheon account has access to [Multidev](/multidev), create a Multidev to push your new code to:
 
 ```bash{promptUser:user}
-composer update drupal/core --with-dependencies
+git push origin composerify && terminus env:create $site.dev composerify
 ```
 
-## Prepare to Deploy
+This will set up the Multidev environment to receive and demo our changed code.
+## Commit
```

Comment:
This section ("Commit") may be removed.

If the user has been following along with the instructions in this document, then all of the work should have been done on the "composerify" branch, and at this point everything should be committed and pushed.